puppet/Puppetfile_forge

```diff
@@ -16,9 +16,10 @@ mod 'puppet/jenkins', '2.0.0'
 mod 'puppet/letsencrypt', '5.0.0'
 mod 'puppet/nodejs', '8.0.0'
 mod 'puppet/selinux', '3.1.0'
-mod 'puppet/unattended_upgrades', '5.1.0'
+mod 'puppet/unattended_upgrades', '6.0.0'
 mod 'puppetlabs/apache', '5.4.0'
 mod 'puppetlabs/apt', '7.7.0'
+mod 'puppetlabs/augeas_core', '1.1.2'
```

Comment:
This should only be a testing dependency. All _core modules are bundled in the agent packages (but not the gem)

Reply:
but we also have this here:

https://github.com/theforeman/foreman-infra/blob/1c15a1d3fa616a316795b38b6a7dfeae3cde223e/puppet/Puppetfile_forge#L27

Reply:
That is probably an oversight.

Reply:
updated